src/psyclone/tests/psyir/tools/dependency_tools_test.py

```diff
@@ -246,24 +339,28 @@ def test_scalar_parallelise(declaration, variable, parser):
     prog = parser(reader)
     psy = PSyFactory("nemo", distributed_memory=False).create(prog)
     loops = psy.invokes.get("test").schedule
-    dep_tools = DependencyTools(["levels", "lat"])
+    dep_tools = DependencyTools(["levels", "lat"],
+                                language_writer=FortranWriter())
+
+    # Test if supplying the loop name as symbol also works:
```

Comment:
What does "also works" mean here? I thought we now always have to supply a symbol?

Reply:
Good point, I forgot that I had changed this consistently everywhere. I then decided to also update the information that declared these arguments in several places to be a `string` still 😉